examples/js_of_ocaml/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Js_of_ocaml • TodoMVC</title>
+		<link rel="stylesheet" href="node_modules/todomvc-common/base.css">
+		<link rel="stylesheet" href="node_modules/todomvc-app-css/index.css">
+	</head>
+	<body>
+		<div id="todomvc"></div>
```

Comment:
it looks like you are using the 2.0 spec however this root element is an ID, can you switch it to a class?